airbyte-integrations/connectors/source-kyriba/unit_tests/test_source.py

```diff
@@ -6,12 +6,6 @@
 
 from source_kyriba.source import KyribaClient, SourceKyriba
 
-config = {
-    "username": "username",
-    "password": "password",
-    "domain": "demo.kyriba.com",
-    "start_date": "2022-01-01",
-}
```

Comment:
I just noticed that this was dead code since it's duplicated below, so deleted it.